### PR TITLE
Ensure correct compliance icon, 7 day timeline, last hour utilization

### PR DESCRIPTION
**Pull request by AllenBW**
https://bugzilla.redhat.com/show_bug.cgi?id=1516807

**tl;dr**

* timeline, when empty shows 7 day range, set default timeline option to a 7 day period, this is then updated to better fit the events data 
* compliance icon is now correct, value is capitalized 
* utilization data now grabs the hour behind the latest (where as before it was grabbing the second item in an array), Ends up vm metrics stream sometimes returns two entries per hour, that means we need to find the one from the last hour, hence the .find 
* utilization available number has been truncated to three places  

### with events in timeline (after)
screen shot 2017-11-28 at 5 39 04 pm

### with events in timeline (before)
screen shot 2017-11-28 at 5 47 51 pm

<hr>

### without events in timeline (after)
screen shot 2017-11-28 at 5 38 48 pm

### without events in timeline (before)  (you'll notice we're missing storage because we were grabbing the wrong hour)
screen shot 2017-11-28 at 5 48 29 pm

### CODE REVIEW

**Comment by AllenBW** [comment]
cc @Loicavenel  lemme know if this satisfies the ask of the bz 🙇‍♀️ ❤️

**Comment by simaishi** [comment]
@AllenBW There is a conflict backporting this to Gaprindashvili for `manageiq-ui-service.pot`. Normally translation related files are not backported and it has its own branch specific PRs.  What should I do with this conflict?

cc @mzazrivec

**Comment by AllenBW** [comment]
@simaishi uggghhh yeah that didn't need to be in this pr, must have missed it, can that file be disregarded  and the rest backported?

**Comment by mzazrivec** [comment]
Changes in `manageiq-ui-service.pot` should not be backported (i.e. the one already in gaprindashvili
branch should stay as it is).

**Comment by simaishi** [comment]
OK, will backport the rest.

**Comment by simaishi** [comment]
Gaprindashvili backport details:

```
$ git log -1
commit d12281acb8ce63bc7511c8cdb01ca3f53a7293bb
Author: Chris Kacerguis <chriskacerguis@users.noreply.github.com>
Date:   Tue Nov 28 19:22:53 2017 -0600

    Merge pull request #1284 from AllenBW/bug/master/#1516807-update-resource-details
    
    Ensure correct compliance icon, 7 day timeline, last hour utilization
    (cherry picked from commit c60105b1dd7f9c862c76e0134bb2b4d176db236b)
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1518748
```
